### PR TITLE
gcbmgr: add list job documentation

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this?
 /kind documentation

#### What this PR does / why we need it:
Add documentation regarding the list jobs in gcbmgr

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by cpanato** [comment]
@justaugustus can you take a look and approve if you are fine if that? thanks!

File: docs/krel/gcbmgr.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 - [Installation](#installation)
 - [Usage](#usage)
 - [Important notes](#important-notes)
+- [List Jobs](#alpha)
```
Comment (justaugustus) [inline docs/krel/gcbmgr.md:12]:
```suggestion
- [List Jobs](#list-jobs)
```

Comment (cpanato) [reply docs/krel/gcbmgr.md]: done. ptal

**Pull request merged**